src/setupProxy.js

```diff
@@ -8,6 +8,11 @@ module.exports = (app) => {
       changeOrigin: true,
       secure: false,
       ws: false,
+      onProxyReq: (proxyReq, req, res) => {
+        if (req.headers['x-jm-authorization']) {
+          proxyReq.setHeader('authorization', req.headers['x-jm-authorization'])
```

Comment:
```suggestion
          proxyReq.setHeader('Authorization', req.headers['x-jm-authorization'])
```

I know headers are case-insensitive here but IIRC it's common practice to capitalize them.